continuous_integration/environment-3.6.yaml

```diff
@@ -1,58 +0,0 @@
-name: test-environment
-channels:
-  - conda-forge
```

Comment:
We might want to keep this build, but with eg py 3.7, since its testing specific versions of numpy and pandas?

Reply:
Do you think maybe min, max, and upstream is enough for our numpy, pandas matrix?

Reply:
Yeah, that sounds good for the time being. We can add an intermediate numpy / pandas version when add a Python 3.9 CI build

Reply:
Given the number of special cases we are having for different versions of pandas at the moment (due to not catching them in time the last months), I think it would help to keep 3 builds for now (for min, max and one in the middle). At least that helps for fixing all the CI builds in https://github.com/dask/dask/pull/6996

Or could we directly add a 3.9 build instead, to not have to drop a build? (only change python versions) 
Or are there many failures with 3.9?

Reply:
Please see this thread ( https://github.com/dask/dask/pull/7006#discussion_r588802282 ).

Yep Python 3.9 was added in PR ( https://github.com/dask/distributed/pull/4460 )